dotcom-rendering/src/components/TableOfContents.stories.tsx

```diff
@@ -10,9 +10,6 @@ const Wrapper = ({ children }: { children: React.ReactNode }) => {
 			css={css`
 				padding: 20px;
 				max-width: 700px;
-				${from.tablet} {
-					width: 700px;
-				}
```

Comment:
How did we decide to remove this?

Reply:
Now that we have the two stories side by side (light and dark) this breaks the stories with the light variant expanding into the dark. Removing this fixes it and the stories seem fine across multiple breakpoints 👍

Reply:
Ok, we could also have a vertically stacked `splitTheme` if necessary as mentioned in https://github.com/guardian/dotcom-rendering/pull/9269#pullrequestreview-1692307393, but if you've tested at various breakpoints, 👌